frigate/app.py

```diff
@@ -205,6 +206,9 @@ def init_queues(self) -> None:
         # Queue for timeline events
         self.timeline_queue: Queue = mp.Queue()
 
+        # Queue for inter process communication
+        self.inter_process_queue: Queue = mp.Queue()
```

Comment:
This should be updated to use the faster queues that were recently merged.

Reply:
Updated